### DISCUSSION TITLE
Seqr on AWS

### DISCUSSION THREAD

**Discussion by davidfasel** [broadinstitute/seqr#1952] [Q&A]
Our team at Columbia is pretty tied to the AWS ecosystem, but we would love to use seqr.  Is there a plan to eventually port seqr to AWS or is the plan to always rely on GCP?

**Comment by hanars** [comment]
The Broad has no plans to change our deployment, but the open-source installation is platform agnostic, and I know some groups have successfully deployed to AWS. We store some resource files in google cloud that you will need to download and our recommended approach is to use the gsutil CLI, but you can manually copy those files to storage in AWS and reference them without needing to be on GCP

Instructions for installation are here, for loading data you will want to use option #2 "annotate and load on-prem"
https://github.com/broadinstitute/seqr/blob/master/deploy/LOCAL_INSTALL.md
